spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCServiceStatus.java

```diff
@@ -19,5 +19,7 @@
     SC_SERVICE_STOPPED,
     SC_SERVICE_RUNNING,
     SC_SERVICE_STARTING,
```

Comment:
You can remove SC_SERVICE_STARTING.  We want SC_SERVICE_RUNNING and SC_SERVICE_STARTED instead